he.json

```diff
@@ -0,0 +1,297 @@
+        },
+        "nav" : {
+            "home" : "ראשי",
+            "shop" : "חנות",
+            "cosmetics" : {
+                "dropdown" : "אייטמים קוסמטיים",
+                "upcoming" : "בקרוב",
+                "list" : "רשימה",
+                "png" : "פורמט png",
+                "gallery" : "גלרייה",
+                "icons" : "אייקונים",
+                "reminders" : "תזכורות",
+                "history" : "היסטוריית החנות"
+            },
+            "guideDropdown" : "מדריך",
+            "news" : "חדשות",
+            "modes" : "מודים",
+            "randomiser" : "סט אקראי",
+            "about" : {
+                "dropdown" : "אודות",
+                "translate" : "תרגם",
+                "about" : "אודות fnbr.co",
+                "privacy" : "מדיניות פרטיות",
+                "donate" : "תרום",
+                "discord" : "דיסקורד",
+                "twitter" : "טוויטר"
+            },
+            "api" : "api",
+            "account" : {
+                "dropdown" : "חשבון",
+                "account" : "חשבון",
+                "logout" : "התנתק"
+            },
+            "languagePicker" : {
+                "trigger" : "שפה",
+                "title" : "בחר שפה",
+                "currentLanguage" : "אתה משתמש כרגע ב[language]",
+                "error" : "שגיאה בעת הורדת קובץ השפה",
+                "authors" : "תורגם על ידי:",
+                "cta" : "לחץ כאן כדי להחליף ל[language]",
+                "switching" : "מוריד [language]...",
+                "finished" : "השפה שלך שונתה בהצלחה!"
+            }
+        },
+        "home" : {
+            "title-upcoming" : "אייטמים צפויים בקרוב",
+            "title-all" : "כל האייטמים",
+            "error" : "לא ניתן לאחזר פריטים, אנא נסה שוב בעוד כמה רגעים.",
+            "error-too-long" : "הפעולה לוקח זמן רב מהצפוי"
+        },
+        "shop" : {
+            "title" : "סבב נוכחי",
+            "title-featured" : "בולט",
+            "title-daily" : "יומי",
+            "error" : "החנות לא זמינה, נסה שוב בעוד כמה רגעים",
+            "line1" : "הסבב הנוכחי של חנות האייטמים בפורטנייט באטל רויאל - מתעדכן בכל יום [בחצות זמן אוניברסלי מתואם](https://time.is/UTC|tooltip=Current time in UTC|newtab=true).",
+            "line2" : "תוכל לצפות בסבב האייטם שופ הקודם [פה](/shop/yesterday|tooltip=Yesterday's Item Shop).",
+            "line3" : "לחץ על אייטם קוסמטי כדי לראות מידע נוסף אודותיו.",
+            "countdown" : {
+                "title" : "אייטמים חדשים:",
+                "wordyFormat" : "[number] [noun]",
+                "hour-singular" : "שעה",
+                "hours-plural" : "שעות"
+            },
+            "yesterday" : {
+                "title" : "סבב האייטם שופ הקודם"
+            },
+            "vote" : {
+                "title" : "מה אתם חושבים על סבב החנות הנוכחי?",
+                "line1" : "החנות מתאפסת מדי יום בחצות זמן אוניברסלי מתואם.",
+                "line2" : "עיין ב[מדיניות הפרטיות שלנו](/privacy) לפני שאתה מצביע",
```

Comment:
It's not allowed to separate the link name (within the square brackets `[...]`) from the link url (inside the round brackets `(...)`). It must be in this format: `[link name](link url)`

Reply:
the link name and url are not separated! you may be using a text editor that is not supporting right to left languages. You can try copying the line to another editor (;